guides/mn-application-aws-lambda-java-runtime/mn-application-aws-lambda-java-runtime.adoc

```diff
@@ -10,7 +10,7 @@ The generated application contains a `HomeController`. It responds to POST reque
 
 source:HomeController[]
 
-callout:controller[arg0=/]
+* The class is defined as a controller with the @api@/io/micronaut/http/annotation/Controller.html[@Controller] annotation mapped to the path `/`.
```

Comment:
why are you replacing the callout with the sentence? Please, use the callout to avoid having to maintain the same copy in multiple places.

Reply:
Because of the issue with some callouts not used as callouts (essentially they don't actually have a pointer in the code, they are used as unordered list items). This breaks the new syntax, since by adding <.> at the beginning of each callout file the line is automatically rendered as callout. It only affects like 2 guides and I think the solution in this case is to not use the callout macro